src/test/scala/com/netaporter/precanned/FancyDslSpec.scala

```diff
@@ -40,7 +42,8 @@ class FancyDslSpec
   }
 
   "path expectation" should "match path" in {
-    animalApi expect path("/animals") and respond using resource("/responses/animals.json") end()
+    animalApi expect path("/animals") and respond using
+      resource("/responses/animals.json") end(defaultBlockUpTo)
```

Comment:
Why did you need to add `defaultBlockUpTo`?

Reply:
It fails to compile for me with scala 2.13 with empty braces:
```
[error] /pre-canned/src/test/scala/com/netaporter/precanned/FancyDslSpec.scala:139:84: type mismatch;
[error]  found   : Unit
[error]  required: scala.concurrent.duration.FiniteDuration
[error]     availablePortApi expect get and path("/status") and respond using entity("OK") end()
```
I couldn't come up with actual reason why it stopped compiling using scala 2.13 and used timeout explicitly

Reply:
We should mention it the README `fancy` part, that `end()` works only for 2.12 and for 2.13 the example should be different.

Reply:
I found out you can't use call nullary methods with postfix notation: https://github.com/scala/bug/issues/11461,
so I decided to split end into two methods: one accepts custom timeout, the other one uses default timeout and can be called without empty braces. I know it's preferred to call methods with side effects using empty braces, but it's a trade-off to keep fancy dsl dotless.